wraithv2/CHANGEMEconfig_wraithv2.lua

```diff
@@ -22,6 +22,8 @@ local config = {
     ,expiresUid = "expiration"
     -- statuses to flag on when scanned
     ,flagOnStatuses = {"STOLEN", "EXPIRED", "PENDING", "SUSPENDED"}
+    -- Vehicle classes that will NOT get ran through CAD
```

Comment:
Please add this link in your config comment https://docs.fivem.net/natives/?_0x29439776AAA00A62